installer/create-database.php

```diff
@@ -1097,7 +1097,8 @@ function populateDatabase($oDB)
             'validation_key' => 'string(38)',
             'validation_key_expiration' => 'datetime',
             'last_forgot_email_password' => 'datetime',
-            'expires' => 'datetime'
+            'expires' => 'datetime',
+            'status' => 'BOOLEAN DEFAULT TRUE'
```

Comment:
Sadly, this is a keyword in MySQL. Rename to active_status or such?

Reply:
Renamed to user_status